kore/src/Kore/Repl/Data.hs

```diff
@@ -507,3 +500,39 @@ data GraphProofStatus
     | StuckProof [Graph.Node]
     | TrustedClaim
     deriving (Eq, Show)
+
+makeAuxReplOutput :: String -> ReplOutput
+makeAuxReplOutput str =
+    ReplOutput . return . AuxOut $ str <> "\n"
+
+makeKoreReplOutput :: String -> ReplOutput
+makeKoreReplOutput str =
+    ReplOutput . return . KoreOut $ str <> "\n"
+
+runUnifierWithExplanation
+    :: forall m a
+    .  MonadSimplify m
+    => UnifierWithExplanation m a
+    -> m (Either ReplOutput (NonEmpty a))
+runUnifierWithExplanation (UnifierWithExplanation unifier) =
+    either explainError failWithExplanation <$> unificationResults
+  where
+    unificationResults
+        ::  m
+                (Either UnificationOrSubstitutionError ([a], First ReplOutput))
```

Comment:
I know you didn't make this change, but this line looks weird. I think you should move it up in-line with the `m`.